pkg/webui/console/components/device-data-form/index.js

```diff
@@ -483,91 +445,27 @@ class DeviceDataForm extends Component {
           component={Input}
         />
         <Message component="h4" content={m.activationSettings} />
-        <Form.Field
-          title={m.activationMode}
-          disabled={update}
-          name="activation_mode"
-          component={Radio.Group}
-        >
+        <Form.Field title={m.activationMode} name="activation_mode" component={Radio.Group}>
```

Comment:
```suggestion
        <Form.Field title={m.activationMode} name="activation_mode" component={Radio.Group} disabled={!mayEditKeys}>
```
I think it's better to simply disable the selector in that case instead.